.github/workflows/build.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         run: |
           python -m pip install --user mkdocs-material mkdocstrings[python] pymdown-extensions
       - name: Build and deploy site
-        run: mkdocs gh-deploy --force
```

Comment:
Removed since IIRC this was just copy-paste and no one really wanted it? I can undo if we wanna keep it.

Reply:
Isn't this necessary to deploy the site?

Reply:
The diff github shows here is misleading, what the commit removes is the `--force` argument, not the entire deploy